run-tests.py

```diff
@@ -13,6 +13,8 @@
 
 # These paths should all pass PEP-8 checks
 paths_to_check = [
+    'server/pulp/plugins/file/',
```

Comment:
This addition introduces two issues:

    server/pulp/plugins/file/distributor.py:104:17: E265 block comment should start with '# '
    server/pulp/plugins/file/distributor.py:226:13: E265 block comment should start with '# '